Makefile

```diff
@@ -78,7 +78,7 @@ build_unit: $(CARGO_TARGET_DIR)
 # Test build cross-architecture
 # Ref: https://github.com/cross-rs/cross
 .PHONY: build_cross
-build_cross: build_cross.aarch64-unknown-linux-gnu build_cross.arm-unknown-linux-gnueabi
+build_cross: build_cross.arm-unknown-linux-gnueabi
```

Comment:
Are you sure about this?  My understanding (could be wrong) is: `aarch64` means ARMv8 64-bit, and `arm` means ARMv7 32-bit.  My knowledge is based on playing around with Fedora on Raspbery-pi's a few years ago :wink:

Reply:
Raspbery-Pi comment Clarification: Meaning my knowledge is old, possibly irreverent, and should be discarded quickly upon the slightest whiff of being incorrect :smile:

Reply:
yes, so this change is expected, right? We don't need aarch64 in the cross task anymore. We only care about the armv7 one unless we wanna send that off to ec2's armv7 instances?

Reply:
@flouthoc opinions?